gradle.properties

```diff
@@ -4,74 +4,74 @@ group=org.ballerinalang
 version=2201.11.0-SNAPSHOT
 codeName=swan-lake
 
-ballerinaLangVersion=2201.10.0
+ballerinaLangVersion=2201.11.0-20241117-133400-a3054b77
 ballerinaJreVersion=2.0.1
-dependencyJREVersion=jdk-17.0.7+7-jre
+dependencyJREVersion=jdk-21.0.5+11-jre
 specVersion=2024R1
 slf4jVersion=1.7.30
 balstdlibBranch=master
 devIdpUrl=https://dev.api.asgardeo.io/oauth2/token/.well-known/openid-configuration
 
 # Stdlib Level 01
-stdlibIoVersion=1.6.1
-stdlibJavaArraysVersion=1.4.0
-stdlibTimeVersion=2.5.0
-stdlibUrlVersion=2.4.0
-stdlibXmldataVersion=2.8.0
-observeVersion=1.3.0
-stdlibMathVectorVersion=1.0.2
-stdlibLdapVersion=1.0.1
+stdlibIoVersion=1.6.2-20241112-233100-995cf5f
+stdlibJavaArraysVersion=1.4.1-20241112-232800-f3adad7
+stdlibTimeVersion=2.6.0-20241113-073800-201b904
+stdlibUrlVersion=2.4.1-20241113-073900-335ff51
+stdlibXmldataVersion=2.8.1-20241112-235200-752b9bc
+observeVersion=1.4.0-20241113-092000-b83ae74
+stdlibMathVectorVersion=1.0.3-20241112-233200-6a39cd1
+stdlibLdapVersion=1.1.0-20241113-084300-d6cc91d
 
 # Stdlib Level 02
-stdlibAvroVersion=1.0.2
-stdlibConstraintVersion=1.5.0
-stdlibCryptoVersion=2.7.2
-stdlibDataXmldataVersion=1.0.0
-stdlibLogVersion=2.10.0
-stdlibOsVersion=1.8.0
-stdlibProtobufVersion=1.6.1
-stdlibPersistVersion=1.4.0
-stdlibRandomVersion=1.5.0
-stdlibTaskVersion=2.5.0
-stdlibXsltVersion=2.7.0
-observeInternalVersion=1.3.0
+stdlibAvroVersion=1.0.3-20241113-081200-95b592a
+stdlibConstraintVersion=1.6.0-20241113-090900-d276ad5
+stdlibCryptoVersion=2.7.3-20241113-081400-d015a39
+stdlibDataXmldataVersion=1.0.1-20241113-122800-f4e815c
+stdlibLogVersion=2.10.1-20241113-120000-4577868
+stdlibOsVersion=1.8.1-20241113-122000-cca973b
+stdlibProtobufVersion=1.6.2-20241113-122200-13cac06
+stdlibPersistVersion=1.4.1-20241113-122000-306cc63
+stdlibRandomVersion=1.5.1-20241113-122300-1bc770e
+stdlibTaskVersion=2.5.1-20241113-123500-f905281
+stdlibXsltVersion=2.7.1-20241113-123200-11dc649
+observeInternalVersion=1.3.1-20241113-101700-265054d
 
 # Stdlib Level 03
-stdlibCacheVersion=3.8.0
-stdlibFileVersion=1.10.0
-stdlibFtpVersion=2.11.0
-stdlibMimeVersion=2.10.0
-stdlibTcpVersion=1.11.0
-stdlibUdpVersion=1.11.0
-stdlibUuidVersion=1.8.0
+stdlibCacheVersion=3.8.1-20241113-125700-b75a1bf
+stdlibFileVersion=1.10.1-20241113-151700-e1a2e38
+stdlibFtpVersion=2.11.1-20241113-154100-5317889
+stdlibMimeVersion=2.10.2-20241113-154200-d953747
+stdlibTcpVersion=1.11.1-20241113-154600-cf61b56
+stdlibUdpVersion=1.11.1-20241113-154800-7443083
+stdlibUuidVersion=1.8.1-20241113-154400-443c67b
 
 # Stdlib Level 04
-stdlibAuthVersion=2.12.0
-stdlibDataJsondataVersion=0.2.0
-stdlibDataYamlVersion=0.1.0
-stdlibEdiVersion=1.3.0
-stdlibEmailVersion=2.10.0
-stdlibJwtVersion=2.13.0
-stdlibMqttVersion=1.2.0
-stdlibOAuth2Version=2.12.0
-stdlibTomlVersion=0.6.0
-stdlibYamlVersion=0.6.0
+stdlibAuthVersion=2.12.1-20241113-162300-ded40eb
+stdlibDataJsondataVersion=0.3.0-20241114-143900-285d739
```

Comment:
```suggestion
stdlibDataJsondataVersion=0.3.0-20241119-191600-348e6c9
```

Reply:
Fixed in fc222e13178db97b3f9a35effbb13b91845cad54